xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -259,6 +259,12 @@ enum AppAction: CustomLogStringConvertible {
     case failDeleteMemo(String)
     /// Deletion attempt succeeded
     case succeedDeleteMemo(Slashlink)
+    
+    case setSelectedAppTab(AppTab)
+    case requestNotebookRoot
+    case requestFeedRoot
+    
+    case setAppTabs(Bool)
```

Comment:
Nit: Would consider naming this `setAppTabsEnabled`. At first I thought this was the action to set the currently active app tab.